### DISCUSSION TITLE
How to use Emgu CV to read information from a table

### DISCUSSION THREAD

**Discussion by profdeveloper231** [emgucv/emgucv#821] [General]
Is there a way I can use Emgu CV to convert this image into a grid of white lines on black background without the text and then read all the data and store in a multidimensional string array?
![tableonly](https://user-images.githubusercontent.com/56290548/223726975-d0fdd3a5-adc5-4ed7-adce-862859d9c88a.png)

**Comment by emgucv** [comment]
One idea is to use Text Region detection to find all the text area, then use OCR to recognize the text into a table, using the corrdinates of the text area. After that you can replace the text area with the background color.

**Reply by profdeveloper231** [reply]
Thanks for your suggestion. I have code that gets all the boxes with text and then scans each box separately but I have a problem reading the number of rows and columns from the image

**Reply by profdeveloper231** [reply]
I have to get all that information and then use Aspose.Words to reproduce the table in a word file
